app/models/miq_template.rb

```diff
@@ -31,7 +31,7 @@ def scan_via_ems?
   end
 
   def self.eligible_for_provisioning
-    supporting(:provisioning).active
+    where(:type => subclasses_supporting(:provisioning).map(&:name)).active
```

Comment:
I considered using `active.supporting(:provisioning)` but that would have still checked every instance for `#supports?`